maven-extension-rollout/rollout.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+commit_msg="Apply Gradle Enterprise Maven Extension"
+basedir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+repositories=$basedir/repositories.txt
+extensions_xml=$basedir/extensions.xml
+checkout_area=`mktemp -d`
+
+function process_repositories() {
+  num=`wc -l $repositories | awk '{ print $1 }'`
```

Comment:
Not sure why `| awk ...` is required. `wc -l` should only print one line anyway?

Reply:
Here is the output without the `awk`. Not sure why it does it because if seems to be harder to process but that's the way it is 🤷‍♂️ 
```
❯ wc -l repositories.txt
       2 repositories.txt
```

Reply:
Ah! I usually use `wc -l` with piping stuff into it. Didn't realize that when using it this way it actually writes the filename - which actually makes sense since you could feed it multiple files (eg with`ed -l *.txt)